cnames_active.js

```diff
@@ -465,6 +465,7 @@ var cnames_active = {
   "cyliim": "cyliim.github.io",
   "cyrena": "cyrenajs.github.io/cyrena",
   "cyris": "iCyris.github.io",
+  "Cn404": "Yssscloud.github.io/cn404",
```

Comment:
```suggestion
  "Cn404": "Yssscloud.github.io",
```
You have the `CNAME` file in https://github.com/Yssscloud/Yssscloud.github.io, not a repository named `cn404`, so we don't need the directory here.